code/stable/dblpendulum/.drasil/inverted_seeds.json

```diff
@@ -0,0 +1 @@
ssful_scipy":["symbols"],"summary":["terms"],"surArea":["terms"],"surface":["terms"],"symbol":["terms"],"sysCtxDiag":["references","labelledContent"],"system":["terms"],"systemconstraint":["terms"],"systemcontext":["terms"],"systemdescription":["terms"],"t0_apache":["symbols"],"tOfCont":["terms"],"tOfSymb":["terms"],"tOfUnit":["terms"],"t_ode":["symbols"],"table":["terms"],"task":["terms"],"template":["terms"],"tension":["terms","symbols"],"term":["terms"],"termAndDef":["terms"],"terminology":["terms"],"thModel":["terms"],"theory":["terms"],"theta_1":["terms","symbols"],"theta_2":["terms","symbols"],"threeD":["terms"],"time":["terms","symbols"],"torque":["terms"],"traceyGraph":["terms"],"traceyMandG":["terms"],"traceyMatrix":["terms"],"transpose_numpy":["symbols"],"twoD":["terms"],"twoDMotion":["references","concepts","conceptInstances"],"typUnc":["terms"],"type":["terms"],"uncertainty":["terms"],"uncertaintycolumn":["terms"],"undergraduate":["terms"],"undergraduatedegree":["terms"],"unit":["terms"],"unit_vect":["terms","symbols"],"unlikelyChg":["terms"],"useCase":["terms"],"useCasetable":["terms"],"user":["terms"],"usercharacteristic":["terms"],"userinput":["terms"],"ut_scipy":["symbols"],"v_x1":["terms","symbols","references","labelledContent","generalDefinitions"],"v_x2":["terms","symbols","references","labelledContent","generalDefinitions"],"v_y1":["terms","symbols","references","labelledContent","generalDefinitions"],"v_y2":["terms","symbols","references","labelledContent","generalDefinitions"],"validation":["terms"],"value":["terms"],"variable":["terms"],"vav":["terms"],"vavplan":["terms"],"vector":["terms"],"velocity":["terms","symbols","references","labelledContent","dataDefinitions"],"velocityTM":["theoryModels","references","labelledContent"],"velocityWiki":["references"],"verification":["terms"],"verifyInptVals":["references","concepts","conceptInstances"],"vertical":["terms"],"verticalMotion":["terms"],"verticalacceleration":["terms"],"verticalforce":["terms"],"verticalposition":["terms"],"verticalvelocity":["terms"],"video":["terms"],"videogame":["terms"],"volume":["terms"],"w_1":["terms","symbols"],"w_2":["terms","symbols"],"weight":["terms"],"xAxis":["terms"],"xComp":["terms"],"xConstAccel":["terms"],"xCoord":["terms"],"xDir":["terms"],"xDist":["terms"],"xForce1":["references","labelledContent","generalDefinitions"],"xForce2":["references","labelledContent","generalDefinitions"],"xPos":["terms"],"xScalAcc":["terms"],"xVel":["terms"],"x_numpy":["symbols"],"y0_apache":["symbols"],"yAxis":["terms"],"yAxisDir":["references","concepts","conceptInstances"],"yComp":["terms"],"yConstAccel":["terms"],"yCoord":["terms"],"yDir":["terms"],"yDist":["terms"],"yForce1":["references","labelledContent","generalDefinitions"],"yForce2":["references","labelledContent","generalDefinitions"],"yPos":["terms"],"yScalAcc":["terms"],"yVel":["terms"],"y_ode":["symbols"],"year":["terms"],"zAxis":["terms"],"zComp":["terms"],"zCoord":["terms"],"zDir":["terms"]}
```

Comment:
Could you print these one-per-line instead of as 1 long line? It will make diffs way easier to read!